packages/calendar/README.md

```diff
@@ -0,0 +1,35 @@
+## Description
```

Comment:
We can add more robust docs later...but know it'll be required at some point before a full release.